q2_mystery_stew/plugin_setup.py

```diff
@@ -451,6 +454,30 @@ def register_single_type_tests(plugin, list_of_params):
             )
 
 
+def register_typemap_tests(plugin):
+    input_map, output_map = \
+        TypeMap({SingleInt1: EchoOutput,
+                 List[SingleInt1 | SingleInt2] | List[SingleInt1]: EchoOutput})
+    params = [Parameter('typemap', Parameter.POSITIONAL_OR_KEYWORD,
+              annotation=TypeMap, default=None)]
+
+    func = function_template_1output
+    disguise_function(func, 'typemap', params, 1)
+
+    plugin.methods.register_function(
+        function=func,
+        inputs={'typemap': input_map},
+        parameters=[],
+        outputs=[('output', output_map)],
+        input_descriptions={},
+        parameter_descriptions={},
+        output_descriptions={},
+        name='typemap_test',
+        description=LOREM_IPSUM,
+        examples=[]
```

Comment:
We will want some Usage examples here so that we can test interfaces.

Reply:
Yeah I wasn't sure how to make one work. I think I did it?